aggregator/orderbook.go

```diff
@@ -28,19 +33,29 @@ func (ob *OrderBook) GetOrders() []*Order {
 	return ob.orders
 }
 
+// GetOrder
+func (ob *OrderBook) GetOrder(orderId uint32) *Order {
+	for _, order := range ob.orders {
```

Comment:
Enhancement: store mapping and using orderId as key for lookup. Array could just store orderId but no need to use it for lookup